src/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp
+{
+   internal class Program
+   {
+      static void Main(string[] args)
+      {
+         Console.WriteLine("Dummy Message");
```

Comment:
We might as well make the text say what the purpose of the project is. Describe that there's no functionality, the project is used to resolve dependencies for R or something.